src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -204,12 +207,13 @@ public class CoreclrTestWrapperLib
         public const string COLLECT_DUMPS_ENVIRONMENT_VAR = "__CollectDumps";
         public const string CRASH_DUMP_FOLDER_ENVIRONMENT_VAR = "__CrashDumpFolder";
 
-        static bool CollectCrashDump(Process process, string path)
+        static bool CollectCrashDump(Process process, string crashDumpPath, StreamWriter outputWriter)
         {
             string coreRoot = Environment.GetEnvironmentVariable("CORE_ROOT");
             string createdumpPath = Path.Combine(coreRoot, "createdump");
-            string arguments = $"--name \"{path}\" {process.Id} --withheap";
+            string arguments = $"--name \"{crashDumpPath}\" {process.Id} --withheap";
             Process createdump = new Process();
```

Comment:
Create a startInfo then pass it to the process, rather than mutating the one that's attached.

Reply:
This code has not been touched in this PR. I would keep it that way for this PR.